### PR TITLE
DK29qE-genesis-validator-data

### PR DESCRIPTION
**Pull request by raj-shekhar1**

### CODE REVIEW

**Comment by raj-shekhar1** [comment]
@spenot09, looks good.
``` 
./ethkey verifypop 0x75474aC55768fAb6fE092191eea8016b955072F5 enode://772248dfe1af5f77e0efc0510e83364bfad55cbd6d3e276f3bd0b4ddec6472aa98645655fd80bbf049ba3da18d219ab30a68fcb98da8e06dd42863dd0356cc95@35.242.168.170:30303 0x6c5AE53a803796D788E917D1fE919BfC8B56d2E6 0xa3aa75e42e99275f7d7985538fedc06e7f128b138a5311702afc0dc129484763645c40c36fdd97ff0d0293b00a031714 0x69871c12787cff4e29160785c84fdd7cd5e68b21b629856271f8c8dc070acaca6cfc0357e8de2797cd48118291f7a0c10bb97e58218c1673f2ba1c93be4566c001fd6ef425ddde2d4601808bd3f67e9dcae38dacf2f6eb3e266d4fdb1100f4b943642bc257de355586c8504ba5ee000f04c6cfaaef72350be6242bddfff02dd4be01969188d7ac0088c44ee03fe5feda0fe7e2f023931040c0cbab0dd094d1005484b888ab44023980fa367dece2c9eed773006bf70f72e82369562547c4fa840a7eca2aa13de8146fb981c770dc250ae4468c13437876dd689e4641456bdcdfd9d9
===== POP Validation Results =====
Node Key:       true
Oracle Key:     true
Consensus Key:  true
```

**Review by spenot09 — APPROVED** [review]

**Comment by spenot09** [comment]
@flofrie could you also approve please? As Raj is the PR author he can't approve it and the merge policy requires two approvals. This is the CTL GV info.

**Review by flofrie — APPROVED** [review]

**Pull request merged**
